pystac_client/client.py

```diff
@@ -78,6 +79,7 @@ def from_file(
 
         return cat
 
+    @lru_cache(1)
```

Comment:
this should probably be more than 1 since it takes the collection id as a parameter

Reply:
Ah right, I remember why I left it off now :-). Any reason _not_ to leave it as the default (i.e. revert it to the one you approved already)?

Reply:
default sounds fine to me